src/feedback-rating/feedback-rating.tsx

```diff
@@ -0,0 +1,44 @@
+import { FeedbackRatingData } from "./feedback-rating-data";
+import { Text } from "src/text";
+import {
+    ChildContainer,
+    Image,
+    MainContainer,
+    SubmitButton,
+} from "./feedback-rating.styles";
+import { StarsContainer } from "./stars-container";
+import { FeedbackRatingProp } from "./types";
+
+export const FeedbackRating = ({
+    imgSrc,
+    buttonLabel,
+    description,
+    ratings,
+    onRatingsChange,
+    onSubmit,
+}: FeedbackRatingProp): JSX.Element => {
+    // =========================================================================
+    // CONST, STATE, REF
+    // =========================================================================
+    const bannerSrc = imgSrc ?? FeedbackRatingData.IMG;
+
+    return (
+        <MainContainer>
```

Comment:
allow `className` prop to be passed in so that the parent can adjust the component margins. can reference smart app banner

Reply:
Updated props and passed in `className` into `MainContainer`